pulp_2_tests/tests/rpm/api_v2/test_download_policies.py

```diff
@@ -300,6 +308,12 @@ def setUpClass(cls):
         api_client.post(download_path, {'verify_all_units': True})
         cls.verified_file_sha = cli_client.run(checksum_cmd).stdout.strip()
 
+    @classmethod
+    def tearDownClass(cls):
+        """Clean up resources."""
+        if cls.repo:
```

Comment:
This expects the `cls.repo` to be successfully created on setup. What about using 

```py
if getattr(cls, 'repo', None):
```